nullaway/src/test/java/com/uber/nullaway/NullAwayJSpecifyGenericsTests.java

```diff
@@ -426,7 +426,7 @@ public void testForMethodReferenceInAnAssignment() {
             "    return o.toString();",
             "  }",
             "  static void testPositive() {",
-            "    // we should report an error here, since Test::foo cannot take",
+            "    // TODO: we should report an error here, since Test::foo cannot take",
```

Comment:
Just as a note, when reading suggestions on code review, it's often useful to consider if the suggestion applies somewhere else in the same PR. I see another two places where these comments should start with `TODO` below. At this point, I can just make that change on the GitHub UI and land, though, or you can do it if you see this comment soon 🙂

Reply:
Updated. Sorry I missed that